cvat-canvas/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "cvat-canvas",
+  "version": "1.0.0",
```

Comment:
Usually for something new I will prefer to have 0.1.0 version. Thus when it is stable and have well-defined stable API we can assign 1.0.0.

Reply:
I will to it

Reply:
Done